EIPS/abstraction.md

```diff
@@ -0,0 +1,93 @@
+### Preamble
+
+    EIP: <to be assigned>
+    Title: Abstraction
+    Author: Vitalik Buterin
+    Type: Standard Track
+    Category: Core
+    Status: Draft
+    Created: 2017-02-10
+    
+### Summary
+
+Implements a set of changes that serve the combined purpose of "abstracting out" signature verification and nonce checking, allowing users to create "account contracts" that perform any desired signature/nonce checks instead of using the mechanism that is currently hard-coded into transaction processing.
+
+### Parameters
+
+* METROPOLIS_FORK_BLKNUM: TBD
+* CHAIN_ID: same as used for EIP 155 (ie. 1 for mainnet, 3 for testnet)
+* NULL_SENDER: 2**160 - 1
+
+### Specification
+
+If `block.number >= METROPOLIS_FORK_BLKNUM`, then:
+1. If the signature of a transaction is `(CHAIN_ID, 0, 0)` (ie. `r = s = 0`, `v = CHAIN_ID`), then treat it as valid and set the sender address to `NULL_SENDER`
```

Comment:
Was `v = CHAIN_ID` really the intention here?
Currently according to https://github.com/ethereum/EIPs/issues/155 `v = CHAIN_ID * 2 + 35` or `v = CHAIN_ID * 2 + 36`
(implemented only as `v = CHAIN_ID * 2 + 35` in cpp-ethereum)

So do we really need additional `v` rule for zero signature instead of following EIP155 here, too?